### PR TITLE
bye bye preliminary - editors

### PR DESCRIPTION
**Pull request by eviljeff**
As much as possible this commit only touches `/editors`

### CODE REVIEW

File: src/olympia/editors/tests/test_helpers.py [hunk 1]
```diff
@@ -471,13 +388,11 @@ def test_logs(self):
...
-        for template in ['nominated_to_nominated', 'nominated_to_preliminary',
+        for template in ['nominated_to_nominated',
                          'nominated_to_public', 'nominated_to_sandbox',
-                         'pending_to_preliminary', 'pending_to_public',
-                         'pending_to_sandbox', 'preliminary_to_preliminary',
+                         'pending_to_public', 'pending_to_sandbox',
                          'author_super_review', 'unlisted_to_reviewed',
-                         'unlisted_to_reviewed_auto',
-                         'unlisted_to_sandbox']:
+                         'unlisted_to_reviewed_auto', 'unlisted_to_sandbox']:
```
Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_helpers.py]: This should at least be a tuple. Sad panda to not being able to use parametrize :(

File: src/olympia/editors/tests/test_helpers.py [hunk 2]
```diff
@@ -493,12 +408,10 @@ def test_notify_email_activity_email(self):
         reply_email = (
             'reviewreply+%s@%s' % (uuid, settings.INBOUND_EMAIL_DOMAIN))
 
-        for template in ['nominated_to_nominated', 'nominated_to_preliminary',
-                         'nominated_to_public', 'nominated_to_sandbox',
-                         'pending_to_preliminary', 'pending_to_public',
-                         'pending_to_sandbox', 'preliminary_to_preliminary',
-                         'author_super_review', 'unlisted_to_reviewed',
-                         'unlisted_to_reviewed_auto',
+        for template in ['nominated_to_nominated', 'nominated_to_public',
```
Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_helpers.py]: Can we make that at least a tuple?

File: src/olympia/editors/tests/test_helpers.py [hunk 3]
```diff
@@ -1146,27 +812,19 @@ def test_pending_to_super_review(self):
...
-                    'Reviewed for status %r, %s()' % (status, process))
+    def test_nominated_review_time_set_version(self):
+        for process in ['process_sandbox', 'process_public']:
+            self.version.update(reviewed=None)
+            self.setup_data(amo.STATUS_NOMINATED)
+            getattr(self.helper.handler, process)()
+            assert self.version.reviewed
+
+    def test_nominated_review_time_set_file(self):
+        for process in ['process_sandbox', 'process_public']:
```
Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_helpers.py]: Another two cases where the list could or should be a tuple since it's not being mutated later on.

**Comment by eviljeff** [comment]
This was part of the work needed for #3088

**Pull request merged**